### PR TITLE
1434 replace alma morph with alma fix

### PR DESCRIPTION
**Pull request by dr0i**
See #1434.

### CODE REVIEW

**Comment by dr0i** [comment]
I had to merge the master and resolved a conflict , see https://github.com/hbz/lobid-resources/pull/1642/commits/cff427dde01ba8f101142c0dbdb9e51418e389f2. This may be wrong.

File: web/conf/context.jsonld [hunk 1]
```diff
@@ -282,6 +282,9 @@
       "@id" : "http://purl.org/ontology/bibo/shortTitle",
       "@container" : "@set"
     },
+    "Sonstiges" : {
+      "@id" : "http://purl.org/lobid/lv#Miscellaneous"
+    },
```
Comment (TobiasNx) [inline web/conf/context.jsonld:287]: Miscellaneous is mapped twice: https://github.com/hbz/lobid-resources/blob/d7c972f12a6b7f4272fb43f8315183e5f360e8a3/web/conf/context.jsonld#L374-L376

**Review by TobiasNx — CHANGES_REQUESTED** [review]

File: web/conf/context.jsonld (hunk 1 above)
Comment (TobiasNx) [reply web/conf/context.jsonld:287]: sonstige is correct

**Comment by TobiasNx** [comment]
Otherwise +1

File: web/conf/context.jsonld (hunk 1 above)
Comment (dr0i) [reply web/conf/context.jsonld]: Good catch ! Fixed with [99b1e54](https://github.com/hbz/lobid-resources/pull/1642/commits/99b1e5488936fadb7df371d5886ed65cb1a92496).

**Comment by dr0i** [comment]
Deployed, closed.

**Pull request merged**